internal/pkg/model/object_test.go

```diff
@@ -148,6 +148,9 @@ func assertDeepEqualNotSame(t *testing.T, a, b interface{}, path string) {
 				valueB.Index(i).Interface(),
 				path+`.`+cast.ToString(i),
 			)
+
+			// Underlying array must be different, check address of the value
+			assert.NotSame(t, valueA.Index(i).Addr().Interface(), valueB.Index(i).Addr().Interface(), path+`.`+cast.ToString(i))
```

Comment:
Test toto NEskopirovanie slice nevedel odhalit.
Ak by tam boli pointre, tak to je pokryte.
Ale ak je tam `string`, tak musime porovna adresy jednotlivych poloziek.

Vid: https://stackoverflow.com/a/53010178